README.md

```diff
@@ -5,6 +5,14 @@
 ### Run Image
 `docker run --rm  -v ./static:/resources -p 8080:8080 uni-luzern:latest`
 
+## Development
+### Hot reload
+To use hot reload start the project with `docker compose up -d`.
```

Comment:
docker compose vs. docker-compose

Reply:
Ich verwende docker compose (wie im README beschrieben) und bei mir funktionnierts.
Was meinst du mit diesem Kommentar?

Reply:
https://stackoverflow.com/questions/66514436/difference-between-docker-compose-and-docker-compose